### DISCUSSION TITLE
A Quiet Peace

### DISCUSSION THREAD

**Discussion by awmayer** [CAWSCIT/ca-meeting-finder-issue-tracker#67] [Ideas]
Would it be possible to get the daily reflection from A Quiet Peace put on the app? They have a similar feature on the Everything AA app.

**Comment by Jackson-R2550** [comment]
I’ve been wondering the same thing for awhile now. I find a digital copy easier than carrying the physical copies of the books. Cause then I can open it up at any point through out my day. I think this would be a great idea even if it was like an in app purchase for a digital copy of the book.
